pkg/project/version_bundle.go

```diff
@@ -8,54 +8,17 @@ func VersionBundle(p string) versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "net-exporter",
-				Description: "Update to 1.6.0.",
-				Kind:        versionbundle.KindChanged,
+				Component:   "cluster-operator",
+				Description: "Add resource implementation to update infrastructure reference labels.",
+				Kind:        versionbundle.KindFixed,
 				URLs: []string{
-					"https://github.com/giantswarm/net-exporter/releases/tag/v1.6.0",
+					"https://github.com/giantswarm/cluster-operator/pull/888",
```

Comment:
This looks wrong and I think these changes need a changelog entry. As the operator functionality is changing.